app/base/static/assets/css/material-dashboard.css

```diff
@@ -10442,8 +10443,9 @@ a[aria-expanded="true"] .caret,
   width: calc(100% - 260px);
   transition: 0.33s, cubic-bezier(0.685, 0.0473, 0.346, 1); }
   .main-panel > .content {
-    margin-top: 10px;
-    padding: 30px 15px;
+    margin-top: 30px;
+    /* margin-top: 10px; */
+    /*padding: 30px 15px;*/
```

Comment:
similar, the commented is the same with the new code

Reply:
keep unchanging in the latest PR